src/interactive/repl.ts

```diff
@@ -38,6 +39,65 @@ function startREPLCommand() {
 
     startREPL(false, true)
 }
+async function checkKillPersistentSession() {
+    if (persistentStopWarning || vscode.workspace.getConfiguration('julia').get<boolean>('persistentSession.stopWarning') === false) {
```

Comment:
Would be good to share the result of `vscode.workspace.getConfiguration('julia')` with the code below.